README.md

```diff
@@ -2,6 +2,5 @@
 
 ## Run
 
-From the project root directory `./run.sh [-a|--addressee <text>]`
-
+From the project root directory `./run.sh [-a|--addressee <text>]`[-c|--christmas]
```

Comment:
The backtick should enclose the entire command

Reply:
Should be `From the project root directory `./run.sh [-a|--addressee <text>] [-c|--christmas]`